### PR TITLE
Update Reloop Beatmix 2/4 controller mapping

### PR DESCRIPTION
**Pull request by sblaisot**
Update controller mapping for Reloop Beatmix 2/4 for Mixxx 2.4

- Replace `hostcue_X_enabled` with `hotcue_X_status`
- Replace `engine.connectControl` with `engine.makeConnection`
- Delay controller status request at startup to make it work again
- Replace `track_samples` with `track_loaded`
- Fix jogLed being lit on track unload

### CODE REVIEW

File: res/controllers/Reloop-Beatmix-2-4-scripts.js [hunk 1]
```diff
@@ -19,9 +19,9 @@ const JogFlashCriticalTime = 15; // number of seconds to quickly blink at the en
  **********************************************************************
  * User References
  * ---------------
- * Wiki/manual : http://www.mixxx.org/wiki/doku.php/reloop_beatmix_2
- * Wiki/manual : http://www.mixxx.org/wiki/doku.php/reloop_beatmix_4
- * support forum : http://mixxx.org/forums/viewtopic.php?f=7&t=8428
+ * Wiki/manual : https://github.com/mixxxdj/mixxx/wiki/reloop_beatmix_2
```
Comment (ronso0) [inline res/controllers/Reloop-Beatmix-2-4-scripts.js:22]:
```suggestion
 * Wiki/manual : https://github.com/mixxxdj/mixxx/wiki/Reloop-Beatmix-2
```

**Comment by ronso0** [comment]
Thanks! There are some pre-commit issues. FYI `print()` can be replaced with `console.log()`
(just noticed this, not going to review the code)

**Comment by sblaisot** [comment]
> Thanks! There are some pre-commit issues. 

Thanks, pre-commit issues have been adressed

> FYI `print()` can be replaced with `console.log()` (just noticed this, not going to review the code)

Ok, I changed all `print()` to `console.log()`

File: res/controllers/Reloop-Beatmix-2-4-scripts.js (hunk 1 above)
Comment (JoergAtGithub) [reply res/controllers/Reloop-Beatmix-2-4-scripts.js:22]: Isn't this redundant to the manual link at the begining of the XML file?

Comment (sblaisot) [reply res/controllers/Reloop-Beatmix-2-4-scripts.js]: Probably, but, hey, this comes from 2016 🤷

**Review by Swiftb0y — APPROVED** [review]

**Comment by sblaisot** [comment]
@Swiftb0y @ronso0 Can I merge this PR now ?

**Comment by Swiftb0y** [comment]
yup, Sorry for forgetting about it.

**Pull request merged**